backend/src/main/kotlin/app/ehrenamtskarte/backend/userdata/webservice/UserImportHandler.kt

```diff
@@ -0,0 +1,111 @@
+package app.ehrenamtskarte.backend.userdata.webservice
+
+import app.ehrenamtskarte.backend.common.utils.dateStringToStartOfDayInstant
+import app.ehrenamtskarte.backend.projects.database.ProjectEntity
+import app.ehrenamtskarte.backend.projects.database.Projects
+import app.ehrenamtskarte.backend.userdata.database.UserEntitlementsRepository
+import app.ehrenamtskarte.backend.userdata.exception.UserImportException
+import io.javalin.http.Context
+import org.apache.commons.csv.CSVFormat
+import org.apache.commons.csv.CSVParser
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeParseException
+
+class UserImportHandler {
+
+    private val logger: Logger = LoggerFactory.getLogger(UserImportHandler::class.java)
+
+    fun handle(ctx: Context) {
+        try {
+            val files = ctx.uploadedFiles("file")
```

Comment:
We have to receive also a valid api token (connected to administrators) here for auth https://github.com/digitalfabrik/entitlementcard/issues/1491 and then we can check which project it is, check if this project is allowed (should be added as a flag in config.yml) and pass the projectID, else throw an error. Please add this todo. I think a separate issue to do the api token check would be good to create

Reply:
> I think a separate issue to do the api token check would be good to create

I think it supposed to be this task: https://github.com/digitalfabrik/entitlementcard/issues/1417

Reply:
ah perfect. then just refer to this issue